### DISCUSSION TITLE
Tell me your favorite places in Japan

### DISCUSSION THREAD

**Discussion by Yanaginagina** [NAIST-SE/AttractingContributors#1028] [General]
https://github.com/Yanaginagina/AttractingContributors-Place

Please share with me your favorite places in Japan so I can enjoy them too.

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Reply by Yanaginagina** [reply]
Thank you!
I contributed to your project too!

**Reply by 3207-Rhims** [reply]
Hi, Can you guys contribute in my repo - https://github.com/3207-Rhims/AttractingContributors-Movie/             Thank you

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite place.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by ryosuke-sota** [comment]
I made a PR on your repo.
Could you contribute to mine? 
Here : https://github.com/ryosuke-sota/AttractingContributors.git
Thank you.

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
